keyboards/splitkb/kyria/keymaps/vial/vial.json

```
@@ -0,0 +1,316 @@
+{
+    "lighting": "vialrgb",
```

```suggestion
    "lighting": "none",
```
`vialrgb` isn't going to do much with `RGB_MATRIX` explicitly disabled